bridges/TwitterBridge.php

```diff
@@ -158,7 +158,8 @@ public function collectData(){
 			// extract tweet timestamp
 			$item['timestamp'] = $tweet->find('span.js-short-timestamp', 0)->getAttribute('data-time');
 			// generate the title
-			$item['title'] = strip_tags($this->fixAnchorSpacing($tweet->find('p.js-tweet-text', 0), '<a>'));
+			$item['title'] = htmlspecialchars_decode(
+				strip_tags($this->fixAnchorSpacing($tweet->find('p.js-tweet-text', 0), '<a>')), ENT_QUOTES);
```

Comment:
`htmlspecialchars_decode` should directly be called on `$tweet->find('p.js-tweet-text', 0)` (before calling `$this->fixAnchorSpacing(...)` and `strip_tags(...)`